tests/unittests/BackendTest.cpp

```diff
@@ -53,23 +55,27 @@ TEST(Interpreter, profileQuantizationForANetwork) {
   Node *O = F->createFullyConnected(bindings, "fc", A, 4);
   O = F->createRELU("relu", O);
   O = F->createRegression("reg", O, Ex);
+  F->createSave("ret", O);
 
   LoweredInfoMap loweredMap;
   CompilationContext cctx{&bindings, &loweredMap};
   cctx.precisionConfig.quantMode = QuantizationMode::Profile;
 
   bindings.allocate(A);
   bindings.allocate(Ex);
-  EE.compile(F, cctx);
+  EE.compile(cctx);
+  bindings.allocate(mod.getPlaceholders());
 
   // TODO: Verify histogram itself, for now just verify min and max.
   // Run inference first time and capture tensor stats.
   updateInputPlaceholders(bindings, {A}, {&inputs});
   EE.run(bindings);
+  // Get the new function post compilation.
+  auto CF = mod.getFunctions().front();
```

Comment:
Why do we need to do this? What makes F invalid?

nit: name is confusing since despite being post compilation, this is not a CompiledFunction.

Reply:
In this case F("main") no longer exists, partitioning created a new function "main_part1" can totally enhance the name and comments to explain this better.

Reply:
I cant imagine we're relying on that anywhere, should it be an error or something?

Reply:
> I cant imagine we're relying on that anywhere, should it be an error or something?

If I understand your question correctly, in quantization profiling flow,  we need the updated function for generating quantization info.